contrib/docker/samples/xml/docker-compose.yml

```diff
@@ -0,0 +1,58 @@
+
```

Comment:
Is this file actually a duplicate by accident?

Reply:
Not a duplicate: it maps the `contrib/docker/samples/xml/conf` to the `/etc/caldavd/` **VOLUME**, to show how using XML to define resources is done. (And also how to override CCS configuration in general).
See line 20